src/edit/index.vue

```diff
@@ -0,0 +1,130 @@
+<template>
+  <div class="tce-sprout-video">
+    <element-placeholder
+      v-if="isEmpty"
+      :is-focused="isFocused"
+      :is-disabled="isDisabled"
+      name="Sprout Video component"
+      icon="mdi-video-image"
+      active-placeholder="Use toolbar to upload the video"
+      active-icon="mdi-arrow-up" />
+    <div v-else>
+      <error-message v-if="errorMessage" :message="errorMessage" />
+      <progress-message v-else-if="infoMessage" :message="infoMessage" />
+      <div ref="player" class="player d-flex align-center justify-center"></div>
```

Comment:
Consider abstracting this in a `SproutPlayer` component (along with setting inner html), i.e.
```suggestion
      <sprout-player class="d-flex align-center justify-center" />
```